### DISCUSSION TITLE
effective masses for hole

### DISCUSSION THREAD

**Discussion by shubhamsinghphy** [SMTG-UCL/sumo#233] [General]
Dear team,

I would like to know why effective masses for hole using sumo-bandstats is giving overestimated value.

I was checking for MoS2

Indirect band gap: 1.661 eV
Direct band gap: 1.676 eV
  k-point: [0.33, 0.33, 0.00]
  k-point indices: 199, 200
  Band indices: 13, 14

Valence band maximum:
  Energy: -2.083 eV
  k-point: [0.00, 0.00, 0.00]
  k-point location: GAMMA
  k-point indices: 0, 299
  Band indices: 13

Conduction band minimum:
  Energy: -0.422 eV
  k-point: [0.33, 0.33, 0.00]
  k-point location: K
  k-point indices: 199, 200
  Band indices: 14

Using parabolic fitting of the band edges

Hole effective masses:
  m_h: -3.070 | band 13 | [0.00, 0.00, 0.00] (GAMMA) -> [0.50, 0.00, 0.00] (M)
  m_h: -3.005 | band 13 | [-0.00, -0.00, 0.00] (GAMMA) -> [0.33, 0.33, 0.00] (K)

Electron effective masses:
  m_e: 0.447 | band 14 | [0.33, 0.33, 0.00] (K) -> [0.50, 0.00, 0.00] (M)
  m_e: 0.440 | band 14 | [0.33, 0.33, 0.00] (K) -> [-0.00, -0.00, 0.00] (GAMMA)

Could you please have a look? I have checked the electronic and lattice parameter and it is matching with reported value.

**Comment by utf** [comment]
Can you share the band structure?

**Comment by shubhamsinghphy** [comment]
![1H_phase_Mos2_bandstrcture](https://github.com/SMTG-Bham/sumo/assets/146715719/1b8e93d0-0b7b-43e9-88bd-1d60ef47984a)

**Reply by utf** [reply]
Looking at the band structure, the VBM is at Gamma and the band dispersion from Gamma -> M and Gamma -> K are very flat. For that reason, I don't think sumo is giving the wrong value. The effective masses from K -> Gamma and K -> M look reasonably smaller, but this is not where the VBM is, and so sumo does not report those values.

**Comment by shubhamsinghphy** [comment]
Actually here band gap is giving indirect (I don't know why). the reported value is direct band gap and hole effective mass is bellow 1. I need to check my calculations in details. Thanks for the response.

**Reply by ajjackson** [reply]
It does look *technically* indirect to me: the VBM is at Gamma and the CBM is at K. But the direct gap at K is clearly a very similar energy difference and worth characterising as well.
